src/data/set/basic.lean

```diff
@@ -2657,6 +2657,12 @@ lemma mem_image2_iff (hf : injective2 f) : f a b ∈ image2 f s t ↔ a ∈ s 
 lemma image2_subset (hs : s ⊆ s') (ht : t ⊆ t') : image2 f s t ⊆ image2 f s' t' :=
 by { rintro _ ⟨a, b, ha, hb, rfl⟩, exact mem_image2_of_mem (hs ha) (ht hb) }
 
+lemma image2_subset_left (ht : t ⊆ t') : image2 f s t ⊆ image2 f s t' :=
+image2_subset (subset.refl _) ht
```

Comment:
```suggestion
image2_subset subset.rfl ht
```

Reply:
Aaah, I always work with finsets and `finset.subset.rfl` doesn't exist, so I always get confused.